projects/client/RabbitMQ.Client/src/client/impl/SocketFrameHandler.cs

```diff
@@ -67,10 +67,12 @@ public static async Task TimeoutAfter(this Task task, int millisecondsTimeout)
         }
     }
 
+
```

Comment:
Was this newline added by an automatic tool of some sort?

Reply:
Don't know, actually. Fixed it.